sale_subscription_financial_risk/data/base_automation.xml

```diff
@@ -1,15 +1,18 @@
-<?xml version="1.0" ?>
+<!--  # Copyright (C) 2019 Amplex
+      # Copyright (C) 2019 Open Source Integrators
+      # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
 <odoo>
     <data noupdate="1">
-        <!-- automated action to check a partner record's financial risk status on payment received and suspend/re-activate as needed -->
+
+        <!-- Automated action to check a partner record's financial risk status
+             on payment received and suspend/re-activate as needed -->
         <record id="check_account_service_suspensions" model="base.automation">
             <field name="name">Suspended Services Check - On Payment</field>
             <field name="model_id" ref="model_account_invoice"/>
             <field name="trigger">on_write</field>
             <field name="active" eval="True"/>
             <field name="state">code</field>
             <field name="code">
-record = model.browse(env.context['active_id'])
```

Comment:
For my own education, how are we able to reference 'record' below without declaring it in here?

Thanks!

Reply:
Sorry for cutting off the code here, this is my first time using the Github review system and I didn't grab it all!

Reply:
Check the help tab, it gives you all the variables available. `record` is one of them and does exactly what you did.